iib/workers/tasks/build.py

```diff
@@ -838,7 +850,13 @@ def handle_add_request(
         ``binary_image`` to use.
     :param list deprecation_list: list of deprecated bundles for the target index image. Defaults
         to ``None``.
+<<<<<<< HEAD
```

Comment:
I think something got messed up in the rebase. Could you please fix it

Reply:
sure